code/web3js/web3js_demo/web3-contract-basic-interaction-async-await.js

```diff
@@ -7,14 +7,52 @@
 console.log('Mastering Ethereum - web3.js basic interactions using async/await')
 console.log('Author: Francisco Javier Rojas García - fjrojasgarcia@gmail.com')
 
+
+const optionDefinitions = [
+  { name: 'localRPC', alias: 'l', type: Boolean },
+  { name: 'infuraFileToken', type: String },
+  { name: 'alchemyFileToken', type: String }
+]
+
+const commandLineArgs = require('command-line-args')
+const options = commandLineArgs(optionDefinitions)
+
 var Web3 = require('web3');
 var fs = require('fs')
 
-// Prepare your Infura host url
-var infura_host = "https://kovan.infura.io"
+if (options.infuraFileToken && !options.localRPC) {
```

Comment:
Would it be more readable to just have a 
const fileToken = // infura option, alchemy option 
 and not  include the second branch?

Reply:
Unfortunately we need to read which token it is to figure out how to make the API request!